pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -384,31 +397,40 @@ const MemoryRow = ({ memorySize, memorySizeUnit, nodeMaxMemory, recommendedMemor
 
     return (
         <>
-            <label htmlFor='memory-size' className='control-label'>
-                {_("Memory")}
-            </label>
-            <FormGroup validationState={validationStateMemory} bsClass='form-group ct-validation-wrapper' controlId='memory'>
-                <MemorySelectRow id='memory-size'
-                    value={memorySize}
-                    maxValue={nodeMaxMemory && convertToUnit(nodeMaxMemory, units.KiB, memorySizeUnit)}
-                    initialUnit={memorySizeUnit}
-                    onValueChange={e => onValueChanged('memorySize', e.target.value)}
-                    onUnitChange={value => onValueChanged('memorySizeUnit', value)} />
-                <HelpBlock id="memory-size-helpblock">
-                    {validationStateMemory === "error" && <p>{validationFailed.memory}</p>}
-                    {recommendedMemoryHelpBlock}
-                    {nodeMaxMemory && <p> {cockpit.format(
-                        _("Up to $0 $1 available on the host"),
-                        Math.floor(convertToUnit(nodeMaxMemory, units.KiB, memorySizeUnit)),
-                        memorySizeUnit,
-                    )}</p>}
-                </HelpBlock>
-            </FormGroup>
+            <Expandable id='memory-details'
+                    toggleText={expandedMemory ? 'Hide Memory Details' : 'Show Memory Details'}
```

Comment:
Needs i18n

Reply:
Removed the expanders so this is not relevant any more.